test/api/connectors/ConnectorSpec.scala

```diff
@@ -128,6 +129,17 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
         )
     }
 
+    protected def willGetWithParameters[T](url: String, parameters: (String, String)*): CallHandler[Future[T]] = {
```

Comment:
doesn't need a rename, could just use method overloading

Reply:
Actually, could just add a new param to the same method. It won't have the inline args, but it's less code overall:
```
    protected def willGet[T](url: String, queryParams: Seq[(String, String)] = Nil): CallHandler[Future[T]] = {
      MockedHttpClient
        .get(
          url = url,
          parameters = queryParams,
          config = dummyHeaderCarrierConfig,
          requiredHeaders = requiredHeaders,
          excludedHeaders = Seq("AnotherHeader" -> "HeaderValue")
        )
    }
```
(Also as above, could you call it queryParams rather than parameters, so it's unambiguous :slightly_smiling_face: )

Reply:
updated